.git-blame-ignore-revs

```diff
@@ -33,3 +33,6 @@ d08ede042b74b8199dc748323768227b88efcf7c
 
 # fix indentation in mk-python-derivation.nix
 d1c1a0c656ccd8bd3b25d3c4287f2d075faf3cf3
+
+# fix style in nixos/modules/services/web-apps/lemmy.nix
+c0f0b4ef0bc01e446c94bfe705425eeb138d6b0d
```

Comment:
```suggestion
```
This can only be added afterwards also the changes here done are pretty small

Reply:
It actually works when in the same PR provided we use a merge commit for the PR. But I agree that it might be good to ditch this for now since it wasn't readjusted during the last force-push.

Reply:
It actually works in the same PR provided the PR is merged via a merge commit; the hashes to be included would be those of the constituent non-merge commits. But I agree that it might be good to ditch this for now since it wasn't readjusted during the last force-push.